libs/components/pages/src/lib/modules/link-list/link-list.component.ts

```diff
@@ -1,28 +1,52 @@
-import { Component, Input } from '@angular/core';
+import { NgTemplateOutlet } from '@angular/common';
+import { Component, computed, contentChildren, input } from '@angular/core';
+import { SkyWaitModule } from '@skyux/indicators';
+import { SkyAppLinkModule, SkyHrefModule } from '@skyux/router';
 
 import { SkyPageLink } from '../action-hub/types/page-link';
 import { SkyPageLinksInput } from '../action-hub/types/page-links-input';
+import { LinkAsModule } from '../link-as/link-as.module';
 
+import { SkyLinkListItemComponent } from './link-list-item.component';
+
+/**
+ * A component that displays a list of links on the right side of the page, or
+ * below the page content on mobile devices.
+ * @internal
```

Comment:
Same here, `@internal`.

Reply:
The `@internal` isn't really needed. Removed.